src/Microsoft.Dnx.Tooling/Building/BuildManager.cs

```diff
@@ -225,6 +225,7 @@ private bool BuildInternal(string projectPath)
 
                     if (configurationSuccess)
                     {
+                        var packDiagnostics = new List<DiagnosticMessage>();
```

Comment:
Nope, we aren't good. The only remaining call is to WriteSummary(allDiagnostics) which is a summary view. We still need to write the individual diagnostic messages from the packaging process. Returning this code.